### PR TITLE
User info BL

### PR DESCRIPTION
**Pull request by Gusvarov**
Before we had mocked user info
image
Now it's not mocked
image

### CODE REVIEW

File: src/app/component/user/components/profile/profile-service/profile.service.ts [hunk 1]
```diff
@@ -3,19 +3,38 @@ import { Observable } from 'rxjs';
 import { CardModel } from '@user-models/card.model';
 import { environment } from '@environment/environment';
 import { HttpClient } from '@angular/common/http';
+import { LocalStorageService } from '@global-service/localstorage/local-storage.service';
+
```
Comment (RomanDeveloperAcc) [inline src/app/component/user/components/profile/profile-service/profile.service.ts:7]: Please, remove empty line

File: src/app/component/user/components/profile/profile-service/profile.service.ts [hunk 2]
```diff
@@ -3,19 +3,38 @@ import { Observable } from 'rxjs';
...
 })
 
 export class ProfileService {
 
-  constructor(private http: HttpClient) { }
+  constructor(private http: HttpClient,
+              private localStorageService: LocalStorageService) { }
 
   private backEnd = environment.backendLink;
+  public userId: number;
```
Comment (RomanDeveloperAcc) [inline src/app/component/user/components/profile/profile-service/profile.service.ts:19]: Please, write your variables before constructor in such order: public first, then private

File: src/app/component/user/components/profile/profile-service/profile.service.ts (hunk 1 above)
Comment (Gusvarov) [reply src/app/component/user/components/profile/profile-service/profile.service.ts:7]: fixed

File: src/app/component/user/components/profile/profile-service/profile.service.ts (hunk 2 above)
Comment (Gusvarov) [reply src/app/component/user/components/profile/profile-service/profile.service.ts]: fixed

**Pull request merged**